lib/MqttClient.js

```diff
@@ -419,6 +422,40 @@ MqttClient.prototype.handleCustomCommand = function (message) {
                     console.info("Missing spot_id");
                 }
                 break;
+
+            /**
+             * {
+             *   "command": "go_to",
+             *   "spot_id": "Somewhere"
+             * }
+             */
+            case CUSTOM_COMMANDS.STORE_MAP:
+                if (msg.name) {
+                    MapManager.storeMap(msg.name, function(err, data) {
+                        if (err) { console.info(err); }
+
+                    })
+                } else {
+                    console.info("Missing name");
+                }
+                break;
+
+            /**
```

Comment:
Comment should be edited:
/**
             * {
             *   "command": "load_map",
             *   "name": "Somemapname"
             * }
             */